models/image_segmentation_efficientsam/README.md

```diff
@@ -0,0 +1,37 @@
+# image_segmentation_efficientsam
+
+EfficientSAM: Leveraged Masked Image Pretraining for Efficient Segment Anything
+
+Notes:
+- 
+
+## Demo
+
+### Python
+Run the following command to try the demo:
+
+```shell
+python demo.py --input /path/to/image
+```
+
+### C++
```

Comment:
Do you plan to add C++ demo? If no, I sugguest to comment or remove it.

Reply:
I will remove it first, if I add c++ demo later, I will add it back.